lib/generators/setup_generator.rb

```diff
@@ -176,13 +176,25 @@ def footer_logo
   end
 
   def gems
-    gem 'bootstrap-sass', '~> 3.3.6'
-    gem 'jquery-rails', '~> 4.3'
+    # Replace sass-rails with sassc-rails
+    gsub_file "#{@new_app}/Gemfile", /^(gem 'sass-rails'.*)$/,
```

Comment:
does this fail silently if there is no sass-rails already in the gemfile?

Reply:
I don't remember how gsub_file behaves if there isn't a match

Reply:
I think it would fail silently, but I don't think we'll have to worry about that until Rails 6. I think 5.2.3 still uses `sass-rails` out of the box. Rails 6 will inevitably require a revision of this part of the generator.